node-tests/blueprints/addon-import-test.js

```diff
@@ -16,10 +16,10 @@ describe('Acceptance: ember generate and destroy addon-import', function() {
 
     return emberNew()
       .then(() => emberGenerateDestroy(args, (file) => {
-        var importFile = file('app/addon-imports/foo.coffee');
+        var importFile = file('app/addon-imports/foo.js');
 
         expect(importFile)
-          .to.contain("export { default } from 'my-app/addon-imports/foo'");
+          .to.contain("export { default } from 'my-app/addon-imports/foo';");
 
         expectCoffee(importFile);
```

Comment:
These lines expect that the contents of the file in question is valid CoffeeScript, and that is not relevant for these tests that only involve JavaScript, so it should be removed.